cloudify_cli/commands/audit_log.py

```diff
@@ -31,18 +37,48 @@ def auditlog():
 @cfy.pass_client()
 def list_logs(creator_name,
               execution_id,
+              since,
+              follow,
+              timeout,
               sort_by,
               descending,
               pagination_offset,
               pagination_size,
               logger,
               client,
               ):
+    since_timestamp = before_to_utc_timestamp(since) if since else None
+    if follow:
+        raise CloudifyCliError('Streaming not available older Python versions.'
+                               '  Consider upgrading your installation to one '
+                               'based on Python>=3.6.')
```

Comment:
```suggestion
        raise CloudifyCliError('Streaming requires Python>=3.6.')
```
look how much shorter it is! :)
but the only reason I touched this, is the "consider". Don't be so soft on users, they have to be beaten into submission, not nicely asked to consider stuff! :P